ArduCopter/control_guided.cpp

```diff
@@ -137,6 +137,10 @@ void Copter::guided_posvel_control_start()
     pos_control->set_xy_target(curr_pos.x, curr_pos.y);
```

Comment:
Please change the commit message of this file to:
Copter: Added altitude controller to guided posvel controller

including sonar alt adjustment

Reply:
done